README.md

```diff
@@ -23,9 +49,23 @@ Here is a small list of notificatio providers Apprise supports:
 
 ## Contributing
 
-Feel free to make a feature request or if you have Python experience; pull requests are welcome too!
+Feel free to make a feature request or if you have Python experience; pull requests are welcome 
+too!
+
+## Troubleshooting
+
+### Could not connect to Docker Engine
+
+Did you mount the Docker socket to the container? Check out the [example](#usage) if you want 
+to know how.
+
+In the special case that you are accessing Docker over TCP, you should place Disrupt in the 
+same network as your TCP socket. Using a Docker socket proxy (like 
+[docker-socket-proxy](https://hub.docker.com/r/tecnativa/docker-socket-proxy/), or 
+[sockguard](https://github.com/buildkite/sockguard)) is recommended for enhanced security 
+in this case.
```

Comment:
Using `docker-socket-proxy` is mainly a security measure used to guard the docker socket from malicious abuse. If the disrupt container gets compromised, having a `docker-socket-proxy` proxy between disrupt and the actual socket will prevent the attacker from executing malicious commands (e.g., spawn a crypto miner container) if the proxy is configured to only allow `GET` requests to `info` and `services`.